daemon/src/routes_taker.rs

```diff
@@ -162,25 +161,22 @@ pub async fn post_cfd_action(
 
 #[derive(Debug, Clone, Deserialize)]
 pub struct WalletRestoreRequest {
-    pub mnemonic: String,
+    pub mnemonic: Mnemonic,
 }
```

Comment:
Can we add a test so we can see what we actually need to pass in? Is it space-separated words?